src/test/java/bio/terra/common/fixtures/ConnectedOperations.java

```diff
@@ -294,16 +294,26 @@ public void deleteTestSnapshot(String id) throws Exception {
         checkDeleteResponse(response);
     }
 
-    public void deleteTestFile(String datasetId, String fileId) throws Exception {
+    public void deleteFile(String datasetId, String fileId) throws Exception {
         MvcResult result = mvc.perform(
             delete("/api/repository/v1/datasets/" + datasetId + "/files/" + fileId))
-                .andReturn();
+            .andReturn();
         logger.info("deleting datasetId:{} objectId:{}", datasetId, fileId);
         MockHttpServletResponse response = validateJobModelAndWait(result);
         assertThat(response.getStatus(), equalTo(HttpStatus.OK.value()));
         checkDeleteResponse(response);
     }
 
+    public ErrorModel deleteFileExpectError(String datasetId, String fileId, HttpStatus expectedStatus)
+        throws Exception {
+        MvcResult result = mvc.perform(
+            delete("/api/repository/v1/datasets/" + datasetId + "/files/" + fileId))
+            .andReturn();
+        logger.info("deleting datasetId:{} objectId:{}", datasetId, fileId);
+        MockHttpServletResponse response = validateJobModelAndWait(result);
+        return handleFailureCase(response, expectedStatus);
+    }
```

Comment:
I don't think this is used anywhere? Maybe left over from our initial debugging?